.dockerignore

```diff
@@ -12,29 +10,37 @@ report.*.json
 # Rust
 packages/backend/native-utils/target
 
-# Coverage
+# Developer stuff
+.git
+Dockerfile
+docker-compose.yml
 coverage
+chart
+doc
+dev
+npm-debug.log
+*.pem
+run.bat
+api-docs.json
+*.log
+*.code-workspace
+*.md
 
 # config
 /.config
+ormconfig.json
 
 # misskey
 built
 db
 elasticsearch
 redis
-npm-debug.log
-*.pem
-run.bat
-api-docs.json
-*.log
-*.code-workspace
 .DS_Store
 files
-ormconfig.json
-packages/backend/assets/instance.css
 
-# dockerignore custom
-.git
-Dockerfile
-docker-compose.yml
+# I don't see why this should be ignored - m.
+# packages/backend/assets/instance.css
+
+
+# temp development
+goblin*.tar
```

Comment:
oops my bad. was just a hack to ship docker images to another server before I had Harbor set up.